tests/app/questionnaire/test_questionnaire_store_updater.py

```diff
@@ -1169,7 +1186,7 @@ def test_dependent_sections_current_section_status_not_updated(mocker):
 
     questionnaire_store_updater.update_section_status = mocker.Mock()
     assert dependent_block_id in progress_store.get_completed_block_ids(
-        *dependent_section_key
+        section_id=dependent_section_key[0], list_item_id=dependent_section_key[1]
```

Comment:
Did these need change as a result of something?

Reply:
Think it was just a consequence of enforcing keyword args